src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -120,7 +120,7 @@ export interface ImportKeyOption {
     knowledgeBaseUri?: string
   }
   required: boolean
-  inputType: OutlinedTextInputProps['keyboardType']
+  inputType: FilledTextInputProps['keyboardType']
```

Comment:
I still think `inputType: 'default' | 'number-pad'` is the correct solution here. There is no reason to support every possible keyboard type.